test/offline/test_td3_bc.py

```diff
@@ -206,8 +206,8 @@ def watch():
         env = gym.make(args.task)
         policy.eval()
         collector = Collector(policy, env)
-        result = collector.collect(n_episode=1, render=args.render)
-        rews, lens = result["rews"], result["lens"]
+        trainer = collector.collect(n_episode=1, render=args.render)
+        rews, lens = trainer["rews"], trainer["lens"]
```

Comment:
same, could you revert it or rename it to collector_result?

Reply:
Done

Reply:
I also removed an unused `watch` function in one of the tests